packages/api/package.json

```diff
@@ -8,14 +8,14 @@
   "scripts": {
     "build": "tsc --project tsconfig.json && tscpaths -p tsconfig.json -s ./src -o ./dist",
     "lint": "eslint . --ext .ts",
-    "start": "ts-node -r tsconfig-paths/register --transpile-only ./src",
-    "start:dev": "nodemon ./src",
-    "test": "ts-mocha --paths -p tsconfig.json src/**/*.spec.ts",
-    "import-praise": "ts-node -r tsconfig-paths/register src/scripts/import-praise.ts",
-    "not-yet-activated": "ts-node -r tsconfig-paths/register src/scripts/not-yet-activated.ts",
-    "replace-quantifier": "ts-node -r tsconfig-paths/register src/scripts/replace-quantifier.ts",
-    "migrator": "ts-node -r tsconfig-paths/register src/scripts/umzug-cli.ts",
-    "drop-database": "ts-node -r tsconfig-paths/register src/scripts/drop-database.ts"
+    "start": "env-cmd --silent -f ../../.env ts-node -r tsconfig-paths/register --transpile-only ./src",
```

Comment:
Why did you take this approach to use `env-cmd` rather than reading env files in each workspace with `dotenv`?

Reply:
When using Docker, we don't want to build on the assumption there being env files. We don't want to place copies of any env files inside the built Docker images. Instead we want the env to be supplied to the container at runtime. In `docker-compose.yml` this is achieved with the `env-file` directive. 

When running the code outside of Docker, env has to be supplied in another fashion, in this case I chose `env-cmd`.

Reply:
> When using Docker, we don't want to build on the assumption there being env files. We don't want to place copies of any env files inside the built Docker images. Instead we want the env to be supplied to the container at runtime. In `docker-compose.yml` this is achieved with the `env-file` directive.
> 
> When running the code outside of Docker, env has to be supplied in another fashion, in this case I chose `env-cmd`.

Hmmm - okay yeah that makes sense. Another option could be to revert back to reading env files with `dotenv` and use a docker volume to make the external `.env` file available within the container. But I can't think of any reason why this wouldn't be fine too.